README.md

```diff
@@ -190,6 +190,7 @@ Gauges:
 * bbb_meetings_listeners - Total number of listeners in all BigBlueButton meetings
 * bbb_meetings_voice_participants - Total number of voice participants in all BigBlueButton meetings
 * bbb_meetings_video_participants - Total number of video participants in all BigBlueButton meetings
+* bbb_meetings_participant_clients - Total number of participants in all BigBlueButton meetings by client
```

Comment:
Maybe add in parenthesis the three possible types (html5, dial-in and flash) to the end of the description.